eventually/authentication/views.py

```diff
@@ -3,19 +3,66 @@
 ============
 """
 
-from utils.utils import json_loads
+
 from django.views.generic.base import View
 from django.contrib.auth import authenticate, login, logout
 from django.http import HttpResponse
+from eventually.settings import FRONT_HOST
+from utils.utils import json_loads
+from utils.jwttoken import create_token, handle_token
+from utils.send_mail import send_email
+from utils.validators import registration_validate
+from authentication.models import CustomUser
 
 
 class UserView(View):
     """UserView view handles GET, POST, PUT, DELETE requests."""
 
 
-def registration_user():
+def registration(request):
     """Registration CustomUser"""
 
+    if request.method == 'POST':
+        data = json_loads(request.body)
+        if not registration_validate(data):
+            return HttpResponse('Data is not valid', status=400)
+
+        user = CustomUser.create(first_name=data['first_name'],
+                                 last_name=data['last_name'],
+                                 middle_name=data['middle_name'],
+                                 email=data['email'].lower().strip(),
+                                 password=data['password'])
+        if not user:
+            return HttpResponse('Email is already exist', status=400)
+
+        ctx = {
+            'first_name': user.first_name,
+            'domain': FRONT_HOST,
+            'token': create_token({'email': user.email}),
+        }
+        message = 'registration'
+        mail_subject = 'Activate account'
+        send_email(mail_subject, message, [user.email], 'registration.html', ctx)
+        msg = 'Please confirm your email address to complete the registration'
+        return HttpResponse(msg, status=201)
+
+
+def activate(request, token):
+    """Activation CustomUser"""
+    if request.method == 'GET':
+        try:
```

Comment:
it is not good practice to use `try` in the views. Choose other function than `objects.get`